src/common/plugin.ts

```diff
@@ -28,7 +29,14 @@ export abstract class Plugin implements IPlugin {
   public hasPermission(message: IMessage): boolean {
     const channelName = this.container.messageService.getChannel(message).name;
     if (typeof this.pluginChannelName === 'string' && this.pluginChannelName !== channelName) {
-      message.reply(`Please use this command in the \`#${this.pluginChannelName}\` channel.`);
+      let channel = '';
+      try {
+        channel = `<#${this.container.guildService.getChannel(this.pluginChannelName).id}>`;
+      } catch (err) {
+        this._errorGen([this.pluginChannelName], err);
+        channel = `\`#${this.pluginChannelName}\``;
+      }
+      message.reply(`Please use this command in the ${channel} channel.`);
```

Comment:
There should never be an error here, since the required channel is hard coded in, and the channel names dont change.
This can just be
```
const id = this.container.guildService.getChannel(this.pluginChannelName);
message.reply(`Please use this command in the <#${id}> channel.`);
```